docs/developer/payments.mdx

```diff
@@ -371,6 +371,10 @@ The response:
 The current status of payment can be determined based on the value of [`transactionEvent.type`](api-reference/payments/enums/transaction-event-type-enum.mdx).
 On some occasions, multiple calls to [`transactionProcess`](api-reference/payments/mutations/transaction-process.mdx) may be required to proceed with the payment.
 
+### Community resources
+
+- [witoszekdev/dummy-payment-server](https://github.com/witoszekdev/dummy-payment-server) - App for testing Transactions API without a real payment provider
```

Comment:
@saleor/apps-guild shouldn't this repository be located in the Saleor org scope?

Reply:
No, we wanted to leave it as a community resource - it's not actively maintained by me - I made it for testing purposes when we were developing changes in dashboard for Transactions API. The app is very bare bones 😅